### PR TITLE
Show profile icons on user list

### PR DESCRIPTION
**Pull request by ohbarye**
## Change

This adds a feature to show GitHub profile icons on the user list. It defaults to your organization logo in case you would add an unknown username (not registered GitHub).

Fancy?

icons

### CODE REVIEW

**Comment by kachick** [comment]
great feature is comming !

**Comment by pankona** [comment]
@ohbarye 

Thank you for your contribution! Welcome to mobu of the world.
The feature looks really great! I will discuss whether pick this feature or not with mobu core members.

**Comment by pankona** [comment]
All core members agreed to pick this great feature! Let's continue to review the changes.
@kachick Please review :muscle:

File: src/atoms/UserProfileIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  user: string;
+};
+
+const UserProfileIcon: FC<UserProfileIconProps> = ({ user }) => (
+  <div className={"user-profile-icon"}>
+    <img
+      src={`https://github.com/${user}.png`}
+      onError={(e: React.BaseSyntheticEvent) => {
+        e.target.onerror = null;
+        e.target.src = "/images/default-profile-icon.png";
```
Comment (pankona) [inline src/atoms/UserProfileIcon.tsx:14]: Nice choice :+1:

**Review by pankona — APPROVED** [review]

**Comment by pankona** [comment]
I've checked it can work on mobile browser. LGTM! :yum:

File: src/atoms/UserProfileIcon.tsx (hunk 1 above)
Comment (kachick) [reply src/atoms/UserProfileIcon.tsx:14]: Yea, the considering fallback tells the author politeness to us 👍 

If there's anything to discuss, it's. This image is good or bad for users 😄 

screen_shot 2021-06-06 14 39 19

File: src/atoms/UserProfileIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import "./UserProfileIcon.css";
+
+type UserProfileIconProps = {
+  user: string;
+};
+
+const UserProfileIcon: FC<UserProfileIconProps> = ({ user }) => (
+  <div className={"user-profile-icon"}>
+    <img
+      src={`https://github.com/${user}.png`}
```
Comment (kachick) [inline src/atoms/UserProfileIcon.tsx:11]: 📝 😮 I didn't know the GitHub feature!

File: src/atoms/UserProfileIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      onError={(e: React.BaseSyntheticEvent) => {
```
Comment (kachick) [inline src/atoms/UserProfileIcon.tsx:12]: I didn't know this React feature. It reminded me it is useful again 😄

https://stackoverflow.com/questions/34097560/react-js-replace-img-src-onerror/48222599#48222599

**Review by kachick — APPROVED** [review]
screen_shot 2021-06-06 14 54 13

Looks perfect! Thanks for your contribution! 🎉 💯

File: src/atoms/UserProfileIcon.tsx (hunk 1 above)
Comment (pankona) [reply src/atoms/UserProfileIcon.tsx:14]: nice picture

**Comment by ohbarye** [comment]
I'm relieved that the core members warmly welcome my suggestion.

BTW, the organization logo seems a kind of controversial topic even among you. I never mind if you'd replace the icon in the future! 😄

**Comment by kachick** [comment]
@ohbarye Hi! This PR is meaningful one, so I have suggested to welcome you as a committer.

screen_shot 2021-06-06 19 25 57

It has been accepted 🎉 

If you feel working on this project is a fun thing, please accept the invitation! 😄 

screen_shot 2021-06-06 19 24 34

**Comment by ohbarye** [comment]
> If you feel working on this project is a fun thing, please accept the invitation! 😄

I'm honored to accept the offer, thank you!

<image src="https://cdn.themix.org.uk/uploads/2016/03/im-just-so-happy-meme.png" />

**Comment by kachick** [comment]
FYI - onerror is **NOT** deprecated

https://github.com/mdn/sprints/issues/4014#issuecomment-1874589996 => https://github.com/mdn/browser-compat-data/pull/21783

**Pull request merged**